misc/keyvaluestorage/src/node-js/level_db.ts

```diff
@@ -4,19 +4,23 @@ export default {
   create(db: string) {
     if(!database) {
       if(db == ':memory:') {
-        const { MemoryLevel } = require('memory-level')
+        const { MemoryLevel } = importDependency('memory-level')
         database = new MemoryLevel({encodingType: 'json'})
       } else { 
-        try { 
-          const Level = require("classic-level")
-          database = new Level(db, { valueEncoding: 'json'})
-        } catch (e) {
-          // User didn't install levelup db, show detailed error
-          throw new Error('To use sign sdk on server side, you need to instal `classic-level`... If you are seeing this error in ssr environment like nextjs or remix and only intend to run sign client in the browser, you can install this as a devDependency to ensure that your builds are passing')
-        }
+        const Level = importDependency("classic-level")
+        database = new Level(db, { valueEncoding: 'json'})
       }
       database.open()
     }
     return database
   },
+}
+
+function importDependency(dependency: string) {
+  try { 
+    return require(`${dependency}`)
+  } catch (e) {
+    // User didn't install levelup db, show detailed error
+    throw new Error(`To use WalletConnect server side, you'll need to install "\`${dependency}\`" dependency . If you are seeing this error during build / in SSR environment, you can add it as a devDependency to make this error go away.`)
```

Comment:
Sorry but still missing articles for the nouns here, I know it's annoying 😂 

```suggestion
    throw new Error(`To use WalletConnect server side, you'll need to install the "\`${dependency}\`" dependency. If you are seeing this error during a build / in an SSR environment, you can add "\`${dependency}\`" as a devDependency to make this error go away.`)
```